oauth2client/contrib/appengine.py

```diff
@@ -49,9 +49,19 @@
 # TODO(dhermes): Resolve import issue.
```

Comment:
Also, to answer your question, no it isn't being addressed. This PR just reduces the amount of conditionally defined code in `oauth2client` and gets 100% code coverage the the `ndb` stuff in the process.

I have no idea why the import failed (`This is a temporary fix for a Google internal issue.`) and don't have a way to fix it now.